webpack/assets/javascripts/react_app/redux/middlewares/IntervalMiddleware/IntervalConstants.js

```diff
@@ -0,0 +1,3 @@
+export const START_INTERVAL = 'START_INTERVAL';
+export const STOP_INTERVAL = 'STOP_INTERVAL';
+export const DEFAULT_INTERVAL = 5000;
```

Comment:
This isn't used anywhere so there is no default value for `polling: true`

Reply:
Good catch! I might dropped it while refactoring.